### PR TITLE
Add more props to input component

### PR DESCRIPTION
**Pull request by beasrodrigues24**
Why:
* In the purposes of doing the sign up page, noticed that it wouldn't be possible to get the information on the box without adding a few props

How:
* By adding onChange, value and an id to the input custom component

### CODE REVIEW

File: frontend/src/components/Input/index.js [hunk 1]
```diff
@@ -2,7 +2,16 @@ import { PropTypes } from "prop-types";
...
 
-const Input = ({ type, placeholder, disabled, error, errorMessage }) => {
+const Input = ({
+  id,
+  type,
+  placeholder,
+  disabled,
+  error,
+  errorMessage,
+  value,
```
Comment (mirhamasala) [inline frontend/src/components/Input/index.js:12]: Why is `value` a prop? Is the parent passing down this?

Comment (beasrodrigues24) [reply frontend/src/components/Input/index.js:12]: The parent will. My idea is that the signup page will have `const [name, setName] = useState("")` and I'll pass value={name} and onChange={(event) => setName(event.target.value)}.

Comment (mirhamasala) [reply frontend/src/components/Input/index.js:12]: Are you trying to pass the `value` back to the parent? [Lifting state up](https://reactjs.org/docs/lifting-state-up.html)?

Comment (beasrodrigues24) [reply frontend/src/components/Input/index.js:12]: Then the same for email I assume

Comment (beasrodrigues24) [reply frontend/src/components/Input/index.js:12]: I decided to push this first before the Signup page to build on a branch created out of this branch

Comment (beasrodrigues24) [reply frontend/src/components/Input/index.js:12]: Yes I think I am (only saw that message now after updating the page )

Comment (mirhamasala) [reply frontend/src/components/Input/index.js:12]: Did you try this?

Comment (beasrodrigues24) [reply frontend/src/components/Input/index.js:12]: Yes, and it worked. Maximino helped me. See [thread](https://subvisualuniverse.slack.com/archives/C03KFP6GQR3/p1659695443983429)

Comment (mirhamasala) [reply frontend/src/components/Input/index.js:12]: Ah, I see!

Comment (mirhamasala) [reply frontend/src/components/Input/index.js:12]: Got it.

**Review by mirhamasala — APPROVED** [review]

**Pull request merged**